scripts/dockerfiles/Dockerfile.cleanbuild

```diff
@@ -11,7 +11,7 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
-FROM golang:1.12
+FROM golang:1.12.4
```

Comment:
should this be `FROM golang:1.15` ?

Reply:
It should be, I will be sending a follow up CR for all the dockerfile updates